### PR TITLE
add profile releaseCN to use local node&cnpm

### PR DESCRIPTION
**Pull request by kimmking**
Changes proposed in this pull request:
- add profile releaseCN to use local node&cnpm

### CODE REVIEW

File: shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/pom.xml [hunk 1]
```diff
@@ -62,6 +62,34 @@
     <profiles>
         <profile>
             <id>release</id>
```
Comment (cherrylzhao) [inline shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-bin-distribution/pom.xml]: release and releaseCN is same, shall we keep only one?

File: shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-src-distribution/pom.xml [hunk 2]
```diff
@@ -44,6 +44,9 @@
     <profiles>
         <profile>
             <id>release</id>
+            <activation>
```
Comment (cherrylzhao) [inline shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-src-distribution/pom.xml]: if we make activation=true distribution will happed every time

Comment (cherrylzhao) [inline shardingsphere-ui/shardingsphere-ui-distribution/shardingsphere-ui-src-distribution/pom.xml]: if we make activation=true distribution will happed every time, I think we should delete it

File: shardingsphere-ui/shardingsphere-ui-frontend/pom.xml [hunk 3]
```diff
@@ -31,63 +31,120 @@
...
+                </goals>
+                <configuration>
+                  <nodeVersion>${node.version}</nodeVersion>
+                  <npmVersion>${npm.version}</npmVersion>
+                </configuration>
+              </execution>
+              <execution>
+                <id>npm run rimraf</id>
+                <goals>
+                  <goal>npm</goal>
```
Comment (terrymanu) [inline shardingsphere-ui/shardingsphere-ui-frontend/pom.xml]: The indent of xml is incorrect.

Comment (kimmking) [reply shardingsphere-ui/shardingsphere-ui-frontend/pom.xml]: done

**Comment by kimmking** [comment]
only add profile in shardingsphere-ui-frontend pom.

**Pull request merged**